lms/static/scripts/frontend_apps/components/test/BookPicker-test.js

```diff
@@ -127,6 +125,18 @@ describe('BookPicker', () => {
     assert.equal(picker.find(buttonSelector).text(), 'Select book');
   });
 
+  it('cancels the dialog when Cancel button clicked', () => {
```

Comment:
The addition of this test was to exercise the new cancel-button click handler, because otherwise it's a coverage miss. The test itself is pretty doopy.